src/test/java/core/basesyntax/strategy/impl/ReturnFruitTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.strategy.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReturnFruitTest {
+    private static final FruitTransaction CORRECT_TRANSACTION
+            = new FruitTransaction(Operation.RETURN, "banana", 20);
+    private static final FruitTransaction CORRECT_FRUIT_THAT_NOT_EXIST
+            = new FruitTransaction(Operation.RETURN, "pineapple", 150);
```

Comment:
again. fruit might not exist in report or in the shop, but customer could return some pineapples that he bought in the past, so your pineapple balance in the shop will increase anyway.

Reply:
and in this case it's ok to receive new fruit, that isn't in storage.